source/documentation/index.md

```diff
@@ -18,20 +18,16 @@ For technical information, see the [CTC Traders API specifications](/api-documen
 ### What have we just released?
 The following is now available to 3rd party developers:
 
-- The XML schemas are available for download [here](https://github.com/hmrc/transit-movements-validator/tree/main/conf/xsd). Links to individual XSD files are available from [NCTS Phase 5 Technical Interface Specification](/guides/ctc-traders-phase5-tis).
-
-    Currently, the IE015 message is the only one that can be tested, using the departure
-endpoint. Other messages will be supported later as the service develops. For this
-reason, these XSDs are issued with the proviso they may be subject to change and
-iterated in the future.
-
+The declaration departure endpoint now accepts messages in JSON format as well as XML format. The JSON schemas are available for download [here](https://github.com/hmrc/transit-movements-validator/tree/main/conf/json).
 
 ### What are we working on now?
-Currently, we are working on enabling the saving of departure declaration information. This allows developers to retrieve departure declaration metadata.
+Currently, we are working on retrieving messages.
```

Comment:
Perhaps "departure metadata and associated messages"? 

I think we need to specify "departures" in here, at least, as that's all we're working on right now.